pallets/parachain-staking/src/weights.rs

```diff
@@ -205,63 +191,62 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainStaking BottomDelegations (r:0 w:1)
 	#[rustfmt::skip]
 	fn join_candidates(x: u32, ) -> Weight {
-		(65_336_000 as Weight)
+		(65_756_000 as Weight)
 			// Standard Error: 1_000
-			.saturating_add((117_000 as Weight).saturating_mul(x as Weight))
+			.saturating_add((73_000 as Weight).saturating_mul(x as Weight))
```

Comment:
Fairly significant change, is this expected?

Reply:
Seems similar to above, only this time `DelegatorState` storage shows up which got a reduction in size but I still believe it's the same problem we have above.